test/preset-mini.test.ts

```diff
@@ -110,6 +110,7 @@ describe('preset-mini', () => {
       'text-opacity-50',
       'text-red',
       'scale-100',
+      'skew-50deg',
```

Comment:
Emm. this case shouldn't be put here. Go the the preset-mini-targets instead.

Reply:
done!